src/protect-function-app/README.md

```diff
@@ -0,0 +1,124 @@
+
+Use these settings in your identity provider configuration.
+
+| Identity provider setting       | Value for this sample app                               | Notes                                                                            |
+|--------------------------------:|:--------------------------------------------------------|:---------------------------------------------------------------------------------|
+| **Identity provider**           | **Microsoft**                                           | Required value for this sample.                                                  |
+| **App registration type**       | **Provide the details of an existing app registration** | Required value for this sample.                                                  |
+| **Application (client) ID**     | `<client-id>`                                           | Required value for this sample. <br/> 'Application (client) ID' of the API's app registration in Azure portal - this value is a GUID     |
+| **Client secret (recommended)** | _None_                                                  | Suggested value for this sample. <br/> This sample doesn't require this feature. |
+| **Issuer URL**                  | `https://login.microsoftonline.com/<tenant-id>/v2.0`    | Required value for this sample. <br/> Update to include 'Tenant ID' of your Azure AD instance - this value is a GUID                     |
+| **Allowed token audiences**     | `api://<client-id>`                                     | Required value for this sample. <br/> 'Application ID URI' of app registration in Azure portal - this value typically starts with api:// |
+| **Restrict access**             | **Require authentication**                              | Required value for this sample.                                                  |
+| **Unauthenticated requests**    | **HTTP 401 Unauthorized: recommended for APIs**         | Suggested value for this sample.                                                 |
+| **Token store**                 | _Unselected_                                            | Suggested value for this sample.                                                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 3. Deploy the Function app
+
+```console
+func azure functionapp publish <your-Function-App-name>
+```
+
+## Access the API
+
+Using Postman, curl, or a similar application, issue an HTTP GET request to https://<your-function-app-name>.azurewebsites.net/api/greeting with an `Authorization` header of `Bearer {VALID-ACCESS-TOKEN}`.
```

Comment:
Without the `\` in there, that whole thing gets treated like an HTML element and isn't rendered on the screen.  I noticed this was broken in the python one as well, and just fixed it there.

```suggestion
Using Postman, curl, or a similar application, issue an HTTP GET request to https://\<your-function-app-name>.azurewebsites.net/api/greeting with an `Authorization` header of `Bearer {VALID-ACCESS-TOKEN}`.
```